app/src/main/resources/queries/subclasses_query.rq

```diff
@@ -0,0 +1,4 @@
+SELECT ?subclass ?subclassLabel WHERE {
+    ?subclass wdt:P279 wd:${QID}.
+    SERVICE wikibase:label { bd:serviceParam wikibase:language {LANG}. }
```

Comment:
Should be `${LANG}` for consistency

Reply:
@nicolas-raoul, thanks I have fixed the issue. Now I am populating the UI with the obtained list